pkg/operators/oci-handler/oci.go

```diff
@@ -96,6 +99,20 @@ func (o *ociHandler) InstanceParams() api.Params {
 			Description: "Secret to use when pulling the gadget image",
 			TypeHint:    api.TypeString,
 		},
+		{
+			Key:          verifyImage,
+			Title:        "Verify image",
+			Description:  "Verify image using the provided public key",
+			DefaultValue: "true",
+			TypeHint:     api.TypeBool,
+		},
+		{
+			Key:          publicKey,
+			Title:        "Public key",
```

Comment:
I think later on we need to support multiple public keys and/or provide a tool to quickly re-sign our binaries.

Reply:
Hmm, I think you're missing the client/server architecture in the picture. "Use `run` as many times as they have different keys" isn't the case there. You would deploy IG and configure it to "only trust gadgets signed by X" and can't change that with just running another process on your client (see above comment regarding global params vs instance params as well.)

So I think configuring it as "only trust IG official gadgets + our own" could be a thing that might be asked for. If we only want to support a single public key at a time, we should at least provide some help explaining how to quickly resign binaries (I'm not suggesting we build our own binary/app for that, but at least docs and maybe a script.)

Reply:
Let's keep it as TODO in an issue for another PR.